lib/ncr/approval_manager.rb

```diff
@@ -0,0 +1,90 @@
+module Ncr
+  class ApprovalManager
+    attr_reader :work_order
```

Comment:
I'm not sure that I've ever seen `attr_reader` (or `attr_accessor`) made private...I can, but I like them specified at the top of the class for visibility reasons. Strong feelings?